src/frontend/context/Vilkårsvurdering/VilkårsvurderingContext.ts

```diff
@@ -79,6 +81,26 @@ const [VilkårsvurderingProvider, useVilkårsvurdering] = constate(({ åpenBehan
                         vilkårType: redigerbartVilkår.verdi.vilkårType,
                     },
                 ],
+                andreVurderinger: [],
```

Comment:
Hvorfor trenger vi dette? Ville rendyrket et nytt endepunkt for denne operasjonen nå som vi utvider RestPersonResultat.

Reply:
Enig i at API til rediger villkår borde modifieres, vi oppretter en Favro oppgave for det.

Reply:
Enig i at API til rediger villkår borde modifieres, vi oppretter en Favro oppgave for det.

https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=Tea-3873